website/index.php

```diff
@@ -3,6 +3,7 @@
 // error_reporting(0);
 
 include 'style/wrapper.inc.php';
+include_once __DIR__ . '/../config/config.inc.php';
```

Comment:
Hm, while you're doing this, can you go and move the config to be the first include in every file that includes it?

This is mostly useful because I have Sentry set up in config, so putting it first allows Sentry to report errors in other includes.

Reply:
Sure thing!